src/Lean/Util/Path.lean

```diff
@@ -10,6 +10,17 @@ with a directory `A/`. `import A` resolves to `path/A.olean`.
 -/
 import Lean.Data.Name
 
+partial def System.FilePath.forEachModuleIn [Monad m] [MonadLiftT IO m]
```

Comment:
Personally speaking I don't like attaching use topic-specific methods to generic types

Reply:
I moved it to `Lean`, like the other functions in the file.